src/wallet/coincontrol.h

```diff
@@ -40,6 +40,8 @@ class CCoinControl
     FeeEstimateMode m_fee_mode;
     //! Minimum chain depth value for coin availability
     int m_min_depth{0};
+    //! Maximum chain depth value for coin availability
+    int m_max_depth{9999999};
```

Comment:
Perhaps for another PR but could be nice to document the "why" behind the value of 9999999 and possibly hoist it to a constant:
```cpp
           src/wallet/coincontrol.h:44:   int m_max_depth{9999999};
           src/wallet/rpcwallet.cpp:2822: int nMaxDepth = 9999999;
           src/wallet/wallet.cpp:2436:    const int max_depth = { coinControl ? coinControl->m_max_depth : 9999999 };
```